client/webserver/site/src/js/markets.ts

```diff
@@ -1219,8 +1219,15 @@ export default class MarketsPage extends BasePage {
         header.expander.classList.remove('ico-arrowdown')
         header.expander.classList.add('ico-arrowup')
       })
-      details.link.href = `order/${ord.id}`
-      app().bindInternalNavigation(div)
+      if (ord.status === OrderUtil.StatusPending) {
+        Doc.show(details.accelerateBttn)
```

Comment:
Did you mean to *hide*? the accelerate button?  There's no swap yet to accelerate if the order is pending.